Makefile

```diff
@@ -6,11 +6,12 @@ help: ## Outputs this help screen
 install: ## Install NodeJS dependencies with Yarn 
 	yarn install
 
```

Comment:
I don't see any command for webapp part of the repository, is it normal ?

Reply:
`yarn install` install everything in the whole workpaces